sim/core/racials.go

```diff
@@ -316,12 +316,10 @@ func applyRaceEffects(agent Agent) {
 			ActionID: actionID,
 			Duration: time.Second * 10,
 			OnGain: func(aura *Aura, sim *Simulation) {
-				character.MultiplyCastSpeed(1.2)
 				character.MultiplyAttackSpeed(sim, 1.2)
 				character.MultiplyResourceRegenSpeed(sim, 1.2)
```

Comment:
Is it the case that effects which are modeled by `MultiplyHaste()` always multiply resource regen speed as well? If so, then you can bundle `unit.MultiplyResourceRegenSpeed()` directly into `unit.MultiplyHaste()`.